### ISSUE TITLE
0.6.0 release missing ... well, stuff

### ISSUE THREAD

**Issue by alerque** [fonttools/skia-pathops#40]
Something is wrong with the 0.6.0 source release. The 0.5.1.post1 release zip file on PyPi was a little over 41 MB. The 0.6.0 release is less than 40 KB.

As expected, it doesn't build:

```
running build_ext
building 'pathops._pathops' extension
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fno-semantic-interposition -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -D_FORTIFY_SOURCE=2 -fPIC -Isrc/cpp/skia-builder/skia -Isrc/cpp/skia-builder/skia/src -I/usr/include/python3.9 -c src/python/pathops/_pathops.cpp -o build/temp.linux-x86_64-3.9/src/python/pathops/_pathops.o -std=c++14
src/python/pathops/_pathops.cpp:655:10: fatal error: include/core/SkPathTypes.h: No such file or directory
  655 | #include "include/core/SkPathTypes.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Either something went wrong with whatever release process was supposed to buindle the relevant Skia sources, or if the process was changed so that these need to be fetched from some other source, how to match them up to the expected sources needs to be documented (yes I know Skia doesn't version their releases in any useful way).

**Comment by anthrotype** [comment]
thanks for catching this, it may have to do with the transitioning from Travis to Github CI. I'll investigate and fix asap

**Comment by anthrotype** [comment]
the CI step where we were creating the sdist was not cloning the repository with --recursive option, thus the skia submodule was empty and the sdist was broken. I fixed that. I'll make a new release shortly

**Comment by anthrotype** [comment]
Published v0.6.0.post1 with fixed sdist package: https://pypi.org/project/skia-pathops/0.6.0.post1/#files
(pypi doesn't let me reupload a file with the same version so I had to bump the local version, but since the code itself hasn't changed I didn't want to bump to 0.6.1).

**Comment by alerque** [comment]
Yes, I saw that thank you. Already bumped the Arch Linux packaging and the build went fine this time.
